conan/internal/runner/ssh.py

```diff
@@ -20,8 +20,11 @@ def ssh_info(msg, error=False):
 class SSHRunner:
 
     def __init__(self, conan_api, command, host_profile, build_profile, args, raw_args):
-        from paramiko.config import SSHConfig
-        from paramiko.client import SSHClient
+        try:
+            from paramiko.config import SSHConfig
+            from paramiko.client import SSHClient
+        except ImportError:
+            raise ConanException("Paramiko is required for SSH runner, try installing it with 'pip install paramiko'")
```

Comment:
I would mention "if Conan is installed in a virtual environment, try to install the paramiko package` 
Bear in mind that Conan can be installed in different ways:
- a binary package from our website (we should make sure those binaries _do_ contain both Docker/paramiko)
- a local virtual env (either manually or via pipx)
- a "system" package manager (e.g. homebrew, a linux distro)

For the first case, users can't do much (we should take care of it ourselves) - for the latter case, `pip install paramiko` may not work or break their system - so I wouldn't make a blanket `pip install paramiko` recommendation

Reply:
I dont know which requirements is the binary package using.
There exist an `extra_requires` on our [setup.py](https://github.com/conan-io/conan/blob/2825235bc8c4118a3ed0fb106df5cfbf51a15baf/setup.py?plain=1#L117) which allow installing more requirements this way: 
```sh
$ pip install conan[runners]
```
I'll change the message